lab-taylor/test/greet-spec.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const assert = require('assert');
+const app = require(__dirname + '/../lib/greet');
```

Comment:
i usualy name the variables that reference my modules after the name of their file,,, that way when the code is complex its easy to tell where things come from